.istanbul.yml

```diff
@@ -1,8 +1,11 @@
 instrumentation:
   root: ./src
-  extensions: 
+  extensions:
     - ".js"
     - ".jsx"
-  excludes: 
-    - "**/node_modules/**"
-    - "**/__tests__/**"
+check:
```

Comment:
it does not pass, there are lots that are not covered here. I've not added any new functionality so I don't want to dive into adding new tests and being responsible for them as part of this pr, though I'm happy to do that next